pkgs/dartpad_ui/web/codemirror/codemirror.js

```diff
@@ -1,3 +1,7 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Can you revert this change and open an issue on the repo to look at setting up a true third_party directory for this repo? We'll need to have a copy-code step (or perhaps, use a symlink?).